src/common/middleware/errorHandler.js

```diff
@@ -1,6 +1,6 @@
 // eslint-disable-next-line no-unused-vars
 export default (err, req, res, next) => {
-  res.status(err.status).send({
+  res.status(err.status || 500).send({
```

Comment:
@Chiazokam can you also remove the `500` here?

Reply:
Alright

Reply:
Done